build/yamls/antrea.yml

```diff
@@ -1283,6 +1283,9 @@ data:
     # Service traffic.
     #  AntreaProxy: true
 
+    # Enable NodePort Service support in AntreaProxy in antrea-agent.
+      AntreaProxyNodePort: true
```

Comment:
I would suggest not to enable it by default. Even we enable it, it does not provide any value, until we have a solution to remove kube-proxy.
@tnqn @antoninbas

Reply:
Agreed

Reply:
This is the plan. I think Weiqiang only enables it for testing. See PR description "To verify this PR, the NodePort support is enabled. The feature gate should be disabled before merging this PR."

Reply:
Yes, it is enabled only for testing. I will disable it when we're ready to merge.